src/acceptance-test/java/net/consensys/orion/acceptance/EthClientStub.java

```diff
@@ -182,6 +183,25 @@ ReceiveResponse receivePrivacy(String digest, String publicKey) {
     return Optional.ofNullable(keyFuture.join());
   }
 
+  public Optional<PrivacyGroup> modifyPrivacyGroup(
+      String address,
+      String from,
+      String privacyGroupId,
+      String endpoint) {
+    ModifyPrivacyGroupRequest modifyPrivacyGroupRequest = new ModifyPrivacyGroupRequest(address, from, privacyGroupId);
+    CompletableFuture<PrivacyGroup> keyFuture = new CompletableFuture<>();
+
+    httpClient.post(clientPort, "localhost", endpoint).handler(resp -> {
+      if (resp.statusCode() == 200) {
+        resp.bodyHandler(body -> keyFuture.complete(deserialize(body, PrivacyGroup.class)));
+      } else {
+        keyFuture.complete(null);
```

Comment:
nit: The test probably needs to confirm not only that Orion returned a non-200 response code, but perhaps literally what the response was? [this might be a later upgrade].

Reply:
I think this would be a good idea for a later upgrade. Just checking for a 200 is in line with what the rest of the methods in this file do, but it's definitely something to look into after demoing add & remove.